packages/styles/src/themes/pentahoPlus.ts

```diff
@@ -730,6 +730,60 @@ const pentahoPlus = makeTheme((theme) => ({
         },
       },
     },
+    HvCard: {
+      classes: {
+        root: {
+          outline: `1px solid ${theme.palette.slate[200]}`,
```

Comment:
I think we should put a `TODO` on the colors from palette we are using to not forget we need to use a token when available.